app/src/styles.sass

```diff
@@ -44,6 +44,13 @@ input
       background-color: tame($userbuttonbgcolor)
       color: tame($useboxfgcolor)
 
+.formactions.login
+  justify-content: center
+  height: 52px
```

Comment:
Since the element is a flex, this changes the geometry of the inner button. This breaks the homogeneity of the application. Either you change the geometry of all the buttons, or you don't change the geometry of this button.

Reply:
But you have said that we can make the buttons in login and signin at center. So the problem is I cannot change the height of the button ?

Reply:
Let me clarify a bit what I mean by "same geometry". Button are stretched horizontally depending on the text they contain. Therefore this direction is more "flexible" than the vertical direction. I think that what constitute the "geometric identity" of buttons are: the height, the vertical padding, the minimal horizontal padding, the border width, and the border corner radius.

Reply:
Thank you for the explication ! Just one question : if I don't change the default settings for the buttons, how can I make it larger as you requested? I am not quite clear.

Reply:
Yes, the problem is to change the height of this button only. Perhaps the buttons are not tall enough for mobiles, but in that case you must change the height of all buttons.

Reply:
OK thank you for this advice ! I will change the .formsation to apply for all the buttons. Do you think that the height is the right size now or I change it more lower ? hahaha

Reply:
Ok. I've many comments:
-  Changing the height of the buttons could be in another PR.
- It would be nice if the buttons were taller only on touch screen displays.
- Check how the height is currently set (line 38 in style.sass). Do not use absolute pixel values.

Reply:
these are good suggestions. I will work on it in a new PR.

Reply:
About your previous question: changing the width of a button is not as dramatic as changing its height, because two buttons don't usually have the same width, depending on the text they contain, but they do have the same height (see for instance the buttons on the present GitHub page).